Button/Button.jsx

```diff
@@ -15,15 +16,21 @@ class Button extends MaterialComponent {
     super();
     this.componentName = "button";
     this._mdcProps = ["dense", "raised", "compact", "unelevated", "stroked"];
+    this.themeProps = ["primary", "secondary"];
   }
   componentDidMount() {
     super.attachRipple();
   }
   materialDom(props) {
     const ButtonElement = props.href ? "a" : "button";
+    let className = "";
+    this.themeProps.forEach(themeProp => {
+      if (themeProp in props) className += generateThemeClass(themeProp);
```

Comment:
If you set an attribute on an element in preact (using Typescript) without defining its value it would default to `true` if the type is a boolean. I would suggest making a check to verify its value is not `false`. Someone might have some code that only sets a button to primary/secondary on certain conditions (see example below). Adding the check should be fairly simple: `if (themeProp in props && props[themeProp] !== false) ...`

__Example:__
```Typescript
const isPrimary = someRandomFunc();

<Button primary={isPrimary}>My button</Button>
```